debian/ola-rdm-tests.rdm_test_server.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Open Lighting Architecture RDM Test Server
+Documentation=man:olad(1)
+After=network.target remote-fs.target olad.service
+Wants=olad.service
+
+[Service]
+User=olad
+Environment=RDM_TEST_SERVER_OPTS="--world-writable"
```

Comment:
It's usually slightly more user friendly to keep this in an EnvironmentFile (which could live under /etc/default), as it is easier to edit that than it is to edit the systemd unit.

See systemd.exec(5) for details on that one.

Reply:
Ah, yeah I see. The init script also read `/etc/default/ola-rdm-tests`. It looks like if I do the following then Systemd will use a default and override it if it is found in the environment file (which matches the init script behavior):
```service
Environment=RDM_TEST_SERVER_OPTS="--world-writable"
EnvironmentFile=/etc/default/ola-rdm-tests
```

Reply:
Fixed (see next line) in file